src/ApiGenerator/Program.cs

```diff
@@ -164,9 +169,57 @@ private static async Task<int> Generate(bool download, string branch, bool inclu
             return 0;
         }
 
+        private static void RunDotNetFormat()
+        {
+            Console.WriteLine();
+            AnsiConsole.Write(new Rule("[b white on chartreuse4] Formatting Code using dotnet format [/]").LeftJustified());
+            Console.WriteLine();
+
+            var rootPath = GetRootPath(typeof(Program).Assembly.Location);
+
+            var relativeFolderList = new List<string>();
+
+            foreach (var item in Generator.ApiGenerator.GeneratedFilePaths)
+            {
+                var relativePath = item.Replace(rootPath.FullName, string.Empty);
+                relativeFolderList.Add($".{relativePath.Replace("\\", "/")}");
+            }
+
+            var si = new System.Diagnostics.ProcessStartInfo
+            {
+                WorkingDirectory = rootPath.FullName,
+                FileName = "dotnet",
+                Arguments = "format -v diag --include " + string.Join(' ', relativeFolderList.Select(item => $"{item}")),
+                RedirectStandardInput = true,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+            };
+
+            var process = System.Diagnostics.Process.Start(si);
+
+            Console.WriteLine();
+            Console.WriteLine($"Running dotnet format --include {string.Join(' ', relativeFolderList.Select(item => $"{item}"))} -v diag");
+
+            // hookup the eventhandlers to capture the data that is received
+            process.OutputDataReceived += (sender, args) => Console.WriteLine(args.Data);
+            process.ErrorDataReceived += (sender, args) => Console.WriteLine(args.Data);
+
+            process.Start();
+
+            // start our event pumps
+            process.BeginOutputReadLine();
+            process.BeginErrorReadLine();
+
+            process.WaitForExit();
+            Console.WriteLine();
```

Comment:
If you add a package reference on `Proc = 0.8.1` which is already used by other projects in the solution, this could be simplified down to something like:
```csharp
            Proc.Exec(new ExecArguments("dotnet", "format", "--verbosity", "normal", "--include", "./src/*/_Generated/")
            {
                WorkingDirectory = GeneratorLocations.SolutionFolder
            })
```

And just define `GeneratorLocations.SolutionFolder` as something like:
```csharp
        public static string SolutionFolder { get; } = $"{Root}../../";
```

Reply:
No problem, thank you for the feedback. I will make the changes this evening

Reply:
PR Comments resolved

Reply:
@Xtansia I updated the pr off the current main branch. If you ignore _generated files, there are less than 20 files to review.